util/dvsim/utils.py

```diff
@@ -528,6 +528,14 @@ def print_msg_list(msg_list_title, msg_list, max_msg_count=-1):
     return md_results
 
 
+def purge_latest_report_dirs(path):
+    ''' Find and remove directories named `latest` under the given path'''
+
+    for subdir, dirs, files in os.walk(path):
+        for dirname in dirs:
+            if (dirname == "latest"):
+                rm_path(os.path.join(subdir, dirname))
+
```

Comment:
Could we call `clean_odirs` with max 14 days instead?

Reply:
The publish_reports.py script right now publish all reports if they found the `latest` directory.
For the purge function here I just want to remove all `latest` directory to prevent two cases:
1). if somehow the dvsim script exit without generate any reports, then the separate publish_report script won't publish the old reports to the cloud again.
2). if in the same branch, previous call run 2 IPs from the private repo and the latest call run different 10 IPs from the public repo, and we only want to publish the results for the second call in the public repo.